lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -0,0 +1,565 @@
+import { Knex } from 'knex';
```

Comment:
data-migration2 should be removed on rds-phase-3. See this commit: 86d4d62805dc627ec585eb6da2b341fc0c841816